CONTRIBUTING.md

```diff
@@ -1,109 +1,218 @@
+
+* Every contribution counts, and will go a long way towards betterment of the code and/or its related files.
+
+## Assistance to get started:page_facing_up:
+
+If you're new to contributing and have no idea about working with repositories, be sure to check [GitHub docs to getting started](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github) from GitHub may also be helpful to refer.
+
+## Essential Guidelines for contribution against Devscript Winter of Code::bookmark_tabs:
+
+Before you start off with the contribution, 
+
+:bulb: Be sure to look into, and adhere to these points: 
+
+* First comment on the opened issues on which you want to work as "Can I take up the work ?","I would like to work on this!",etc.:loudspeaker:
+
+* Wait for ***approval***. After it's assigned to you then move on further, fork the repository and clone it (if haven't yet done) and start your work according to the guidelines given in the [README.md](https://github.com/bhavesh1129/Blogger/blob/master/README.md) of repository.
+
+## Steps to Contribute in GitHub Repository:pushpin:
+
+***If you don't have git on your machine, [install it](https://help.github.com/articles/set-up-git/).***
+
+### 1. Fork this repository 🚀
+
+* Fork this repository by clicking on the fork button on the top of this page.
+<p align="center">
+    <img src="https://raw.githubusercontent.com/bhavesh1129/orphan_support-php/main/assets/img/fork.png"/>
+  </a>
+ 
+* This will create a copy of this repository in your account.
+
+### 2. Clone the repository 🏁
+ 
+* Now clone the forked repository to your machine. Go to your GitHub account, open the forked repository, click on the code button and then click the _copy to clipboard_ icon.
+
+* After cloning the repository, please read and understand what the existing code or document is meant for, before overwriting it with your changes. If you're unsure about any part, feel free to ask it in discussions, and we'd be happy to help.
+<p align="center">
+    <img src="https://raw.githubusercontent.com/bhavesh1129/orphan_support-php/main/assets/img/clone.png">
+  </a>
+ 
+* Open a terminal and run the following git command:
 
 ```
-git clone --depth 1 https://github.com/<your_user_name>/Blogger.git
+git clone "https://github.com/bhavesh1129/Blogger.git"
 ```
-   <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcT5N0HJ9db7jSvcL4dsDscZQBzqQqqKVs0BnO1OVz26glLWKJRY&usqp=CAU" width="300">
-**3.** Navigate to the project directory.
+
+### 3. Navigate to the project directory.⚓
+
+* Change to the repository directory on your computer (if you are not already there):
 
 ```
-cd Blogger/mysite
+cd Blogger
 ```
-**4.** Add a reference to the original repository.
+
+### 4. Add a reference to the original repository.:ledger:
 
 ```
-git remote add upstream https://github.com/harsh-9in/Blogger.git (url of the repository)
+git remote add upstream https://github.com/bhavesh1129-/Blogger.git (URL of the repository)
```

Comment:
I think you have put the wrong link here

Reply:
No it's absolutely right as I had added the folder in my forked repo so it is showing my username in it.
I had corrected the wrong link which I had used in linking your Readme file to your contributing file.